src/main/java/org/folio/dew/error/CommonExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.dew.error;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.dew.service.SaveErrorService;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@Log4j2
+@RequiredArgsConstructor
+@ControllerAdvice
+public class CommonExceptionHandler {
+
+  private final SaveErrorService saveErrorService;
+
+  @ExceptionHandler(Exception.class)
```

Comment:
It's a bit unclear. Are we going to have separate processing of BulkEditException here? This case we need to check if exception is BulkEditException in the defaultErrorHandler and execute needed logic. Otherwise we need to make separate handler only for BulkEditException?

Reply:
I deleted this handler, it was actually redundant.